MANIFEST.in

```diff
@@ -1,5 +1,6 @@
-include dash_bio/bundle.js
 include dash_bio/dash_bio-shared.js
+include dash_bio/dash_bio-shared.js.map
+include dash_bio/bundle.js
```

Comment:
also `bundle.js.map`?

Reply:
Thanks for catching that, added in c9eef3c.

Reply:
?? I don't see a change to `MANIFEST.in` in that commit.

Reply:
Sorry about that, I was working on 2 concurrent branches and added my changes to the other one by mistake. Fixed in 
331217b .